benchmark/child_process/child-process-params.js

```diff
@@ -0,0 +1,115 @@
+'use strict';
+
+const common = require('../common.js');
+const cp = require('child_process');
+
+const command = 'echo';
+const args = ['hello'];
+const options = {};
+const cb = () => {};
+
+const configs = {
+  n: [1e3],
+  methodName: [
+    'exec', 'execSync',
+    'execFile', 'execFileSync',
+    'spawn', 'spawnSync',
+  ],
+  params: [1, 2, 3, 4],
+};
+
+const bench = common.createBenchmark(main, configs);
+
+function main(conf) {
+  const n = +conf.n;
+  const methodName = conf.methodName;
+  const params = +conf.params;
+
+  const method = cp[methodName];
+
+  if (methodName === 'exec') {
```

Comment:
This would be more manageable as a `switch (methodName) {}` structure... with each case handled in a separate function.